### PR TITLE
Direct to className and add test

### PR DESCRIPTION
**Pull request by MarkGotham**

### CODE REVIEW

File: music21/languageExcerpts/instrumentLookup.py [hunk 1]
```diff
@@ -1404,49 +1308,46 @@
...
-        self.assertEqual(allToBestName[russianTest], russianToBestName[russianTest])
-
-        spanishTest = 'acorde\xf3n'
-        self.assertEqual(allToBestName[spanishTest], spanishToBestName[spanishTest])
+    def testAllClassNames(self):
+        '''
+        Test that all class names are real.
+        '''
+        from music21 import instrument as instr
+        for v in allToClassName.values():
```
Comment (jacobtylerwalls) [inline music21/languageExcerpts/instrumentLookup.py:1349]: Nitpick detail, but can these tests use self.subTest? Some examples in the codebase. So we don't have to re-run to get all the failures.

**Comment by mscuthbert** [comment]
it looks really good.  It's not backwards compatible though.  I want to think for a bit if that matters so much though since these dictionaries have never been fully documented.  But they are exposed as part of the public interface, so we might need to merge this into a v.8 branch.  I know I've used the "abbreviationToBestName" in other code, though not the language lookups.

**Comment by MarkGotham** [comment]
> it looks really good. It's not backwards compatible though. I want to think for a bit if that matters so much though since these dictionaries have never been fully documented. But they are exposed as part of the public interface, so we might need to merge this into a v.8 branch. I know I've used the "abbreviationToBestName" in other code, though not the language lookups.

Ok, noted and no problem. I'll leave it with you. I have a couple of final suggestions for this corner of the code base (or rather implementations of what we've discussed wrt this issue) but will hold until you decide on this.

**Comment by mscuthbert** [comment]
I think that it's enough of an improvement that I'm okay with the lack of backwards compatibility in this shadowy part of the code.  Were there other things you wanted to add or shall I merge?

**Comment by MarkGotham** [comment]
Thanks @mscuthbert.
This is complete in itself. 
I have a couple of final suggestions proposed on #1064 but they can be considered (and implemented or otherwise) separately.
So do go ahead and merge.

**Pull request merged**